docs/json_rpc_api.md

```diff
@@ -632,12 +632,12 @@ The following is an overview of the RPC methods which are implemented by btcd, b

```

Comment:
Mind correcting an existing typo in the documentation?

It's currently:
```
"coinbase": "data", (string) the hex-dencoded bytes of the signature script
```

But should be:
```
"coinbase": "data", (string) the hex-encoded bytes of the signature script
```